### PR TITLE
Update golang version to 1.18.3

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
Update golang version to 1.18.3
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module kubevirt.io/containerized-data-importer
 
-go 1.17
+go 1.18
```
Comment (mhenriks) [inline go.mod:3]: Should probably update the go.mod in staging well

Comment (awels) [reply go.mod:3]: Yup done

**Pull request merged**